harper-core/src/linting/pronoun_knew.rs

```diff
@@ -20,7 +20,10 @@ impl Default for PronounKnew {
         let pronoun_pattern = |tok: &Token, source: &[char]| {
             if tok.kind.is_pronoun() {
                 let pronorm = tok.span.get_content_string(source).to_lowercase();
-                return pronorm != "its" && pronorm != "his" && pronorm != "her";
+                // Personal pronouns that are not only/always subject pronouns.
+                return pronorm != "its" && pronorm != "his" && pronorm != "her"
+                    // Indefinite pronouns can come before adjectives.
+                    && pronorm != "every" && pronorm != "something" && pronorm != "nothing";
```

Comment:
Note: you can probably do something like:
```rust
let exclude = ["its", "his", "her", "every", "something", "nothing"];
return !exclude.contains(&pronorm.as_str());
```

Reply:
> Note: you can probably do something like:
> 
> ```rust
> let exclude = ["its", "his", "her", "every", "something", "nothing"];
> return !exclude.contains(&pronorm.as_str());
> ```

I often have to fight with things like that in Rust that are straightforward in other languages due to its three stringish types and two arrayish types. I'm still learning Rust and some things haven't fully sunk in yet. I'll have a quick look now though...